dll/win32/url/CMakeLists.txt

```
@@ -8,7 +8,14 @@ list(APPEND SOURCE
     ${CMAKE_CURRENT_BINARY_DIR}/url_stubs.c
     ${CMAKE_CURRENT_BINARY_DIR}/url.def)
 
```

We put file lists before the build target declaration.
```suggestion

list(APPEND url_rc_deps
    ${CMAKE_CURRENT_SOURCE_DIR}/res/102.ico)

set_source_files_properties(url.rc PROPERTIES OBJECT_DEPENDS "${url_rc_deps}")

```

---

> We put file lists before the build target declaration.

Tell that to ieframe where I copied from...